modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,75 +25,74 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
+. List all of the containers in the `ovnkube-node` pods by running the following command:
 +
 [source,terminal]
 ----
-$ oc get pods ovnkube-master-9g7zt \
--o jsonpath='{.spec.containers[*].name}' -n openshift-ovn-kubernetes
+$ oc get pods ovnkube-node-bcvts -o jsonpath='{.spec.containers[*].name}' -n openshift-ovn-kubernetes
 ----
 .Expected output
 +
 [source,terminal]
 ----
-northd nbdb kube-rbac-proxy sbdb ovnkube-master ovn-dbchecker
+ovn-controller ovn-acl-logging kube-rbac-proxy-node kube-rbac-proxy-ovn-metrics northd nbdb sbdb ovnkube-controller
 ----
-+
-The `ovnkube-master` pod is made up of several containers.
-It is responsible for hosting the northbound database (`nbdb` container), the southbound database (`sbdb` container), watching for cluster events for pods, egressIP, namespaces, services, endpoints, egress firewall, and network policy and writing them to the northbound database (`ovnkube-master` pod), as well as managing pod subnet allocation to nodes.
+The `ovnkube-node` pod is made up of several containers. It is responsible for hosting the northbound database (`nbdb` container), the southbound database (`sbdb` container), the north daemon (`northd` container), `ovn-controller` and the `ovnkube-controller`container. The `ovnkube-controller` container watches for API objects like pods, egressIPs, namespaces, services, endpoints, egress firewall, network policy etc. It is also responsible for allocating pod IP from the available subnet pool for that node.
 
-. List all the containers in the `ovnkube-node` pods by running the following command:
+. List all the containers in the `ovnkube-control-plane` pods by running the following command:
 +
 [source,terminal]
 ----
-$ oc get pods ovnkube-node-jg52r \
--o jsonpath='{.spec.containers[*].name}' -n openshift-ovn-kubernetes
+$ oc get pods ovnkube-control-plane-65c6f55656-6d55h -o jsonpath='{.spec.containers[*].name}' -n openshift-ovn-kubernetes
 ----
 .Expected output
 +
 [source,terminal]
 ----
-ovn-controller ovn-acl-logging kube-rbac-proxy kube-rbac-proxy-ovn-metrics ovnkube-node
+kube-rbac-proxy ovnkube-cluster-manager
 ----
 +
-The `ovnkube-node` pod has a container (`ovn-controller`) that resides on each {product-title} node. Each node’s `ovn-controller` connects the OVN northbound to the OVN southbound database to learn about the OVN configuration. The `ovn-controller` connects southbound to `ovs-vswitchd` as an OpenFlow controller, for control over network traffic, and to the local `ovsdb-server` to allow it to monitor and control Open vSwitch configuration.
\ No newline at end of file
+The `ovnkube-control-plane` pod has a container (`ovnkube-cluster-manager`) that resides on each {product-title} node. The `ovnkube-cluster-manager` container allocates pod subnet and join subnet IP to each node in the cluster. The `kube-rbac-proxy` container monitors metrics for the `ovnkube-cluster-manager` container.
```

Comment:
Not sure if "join subnet" is a typo or just networking jargon I don't know :)

Reply:
join subnet probably means default for e.g 169.254.169.0/29 and user configured say 100.64.0.0/16 would be joined and routable among each other. cc @tssurya to confirm

Reply:
Discussed over slack and updated.